commitlint.config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
`.json` because cosmiconfig (commitlint's config loader) doesn't load `.js` in an ESM project, and it appear to permit `.cjs`. See cosmiconfig/issues#224.